src/Sulu/Bundle/CoreBundle/Tests/Functional/Controller/LocalizationControllerTest.php

```diff
@@ -11,25 +11,14 @@
 
 namespace Sulu\Bundle\CoreBundle\Tests\Functional\Controller;
 
-use Doctrine\ORM\EntityManager;
 use Sulu\Bundle\TestBundle\Testing\SuluTestCase;
 
 class LocalizationControllerTest extends SuluTestCase
 {
-    /**
-     * @var EntityManager
-     */
-    private $em;
-
-    public function setUp(): void
-    {
-        $this->em = $this->getEntityManager();
-        $this->purgeDatabase();
-    }
-
     public function testCgetAction()
     {
         $client = $this->createAuthenticatedClient();
+        $this->purgeDatabase();
```

Comment:
Shouldn't we keep that in the `setUp`, in case there'll be more tests added?

Reply:
Did move it to the setup.